packages/playground/src/app.jsx

```diff
@@ -9,6 +9,7 @@ import isEqualWith from 'lodash/isEqualWith';
 
 import DemoFrame from './DemoFrame';
 import ErrorBoundary from './ErrorBoundary';
+import { GeoPosition, CopyLink, ThemeSelector } from '~/components';
```

Comment:
I'm not a fan of the `~/` notation in this case. Why not just use `./components`? That way you don't need the mapping of `~` in the `tsconfig.json` and `vite.config.ts`